### PR TITLE
[Internal] Pull Request: Fix the pull request title validation duplicates

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template

## Description

Updated to use latest version the PR title linter. This uses a github request change comment to block the PR rather than the check status. 

Added v4 tag and refactor to the regex.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### CODE REVIEW

File: .github/workflows/prlint.yml [hunk 1]
```diff
@@ -1,15 +1,93 @@
...
+            started_at: new Date(),
+            output: {
+              title: 'Checking the pull request title',
+              summary: `Determining if the title, ${title}, is valid.`
+            }
+          })
+          
+          console.log(checkRun) // <= Is `id` defined in this payload?
+          // https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines
+          const ANGULAR_REGEX = '(\[Internal\] )?.{3}.+: (Add|Fix) .{3}.+'
```
Comment (ealsur) [inline .github/workflows/prlint.yml:70]: Rename?

File: .github/workflows/prlint.yml [hunk 2]
```diff
@@ -1,15 +1,93 @@
...
+            head_branch: headBranch,
+            status: 'in_progress',
+            started_at: new Date(),
+            output: {
+              title: 'Checking the pull request title',
+              summary: `Determining if the title, ${title}, is valid.`
+            }
+          })
+          
+          console.log(checkRun) // <= Is `id` defined in this payload?
```
Comment (ealsur) [inline .github/workflows/prlint.yml:68]: These console.log are needed? Or are just for debugging purposes?

File: .github/workflows/prlint.yml [hunk 3]
```diff
@@ -1,15 +1,93 @@
...
+          // https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-commit-message-guidelines
+          const ANGULAR_REGEX = '(\[Internal\] )?.{3}.+: (Add|Fix) .{3}.+'
+          const regex = new RegExp(ANGULAR_REGEX, 'i')
+          
+          let titleValidity = false
+          let message = `The PR title, ${title}, does not match the regex, ${ANGULAR_REGEX}.`
```
Comment (ealsur) [inline .github/workflows/prlint.yml:74]: We can use a similar message to what we have right now? Not everyone talks regex :)

File: .github/workflows/prlint.yml (hunk 2 above)
Comment (j82w) [reply .github/workflows/prlint.yml:68]: Just for debugging purposes.

**Pull request merged**